Remotion/Data/DomainObjects/DomainObject.cs

```diff
@@ -170,7 +170,7 @@ public class DomainObject : IDomainObject
     private ObjectID _id;
     private ClientTransaction _rootTransaction;
     private bool _needsLoadModeDataContainerOnly; // true if the object was created by a constructor call or OnLoaded has already been called once
-    private bool _isReferenceInitializeEventExecuting; // true only while OnReferenceInitializing is executed
+    private DomainObjectTransactionContextStrategy? _strategy;
```

Comment:
The field name cannot stay as is, `_strategy` is not very helpful. We should change that once we decided on the names.